flutter/flutter.mk

```diff
@@ -138,6 +138,7 @@ flutter/l10n:
 
 .PHONY: flutter/pub
 flutter/pub:
+	git config --global --add safe.directory '/image-workdir/flutter'
```

Comment:
I see that this PR is not yet ready but please let's not add CI-specific actions into the makefile target that is supposed to run everywhere.